<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cancelado</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="tela_cancelar">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>-5</y>
      <width>410</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 43, 43);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="cancelar_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>400</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(31, 106, 165);
background-color: rgb(43, 43, 43);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;SAIR DO WINDOWS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="cancelar_2">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>90</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(230, 212, 9);
font-weight: bold;
font-size: 30px</string>
     </property>
     <property name="text">
      <string>CANCELADO!</string>
     </property>
    </widget>
    <widget class="QLabel" name="cancelar_4">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>170</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(36, 147, 58);
font-weight: bold;
font-size: 30px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../AppData/Local/Programs/Python/Python311/Lib/site-packages/qt6_applications/Qt/bin/icons.qrc">:/icons/Downloads/icons8-atenção-96.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="cancelar_1">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>300</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 25px;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Operação cancelada &lt;br/&gt;pelo usuário!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ok">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>440</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(31, 106, 165);
color: rgb(255, 255, 255); 
border-radius: 10px;
font-weight: bold;
font-size:15px;</string>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../../AppData/Local/Programs/Python/Python311/Lib/site-packages/qt6_applications/Qt/bin/icons.qrc"/>
 </resources>
 <connections/>
</ui>
